wordpress-mobile/WordPress-iOS
# Issue/subdomains

Fixes #10679

To test:
1. Go through the site creation process with a blog site type selected. You should see .blog subdomains in the suggested domains.
2. Go through the site creation process with the other two site types selected. You shouldn't see any .blog subdomains.

Update release notes:
